configs/log_test_user.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
What is this config supposed to represent?

Reply:
this must be something left-over from tests. let me remove it